CHANGELOG.asciidoc

```diff
@@ -7,6 +7,52 @@
 === Beats version 8.4.1
 https://github.com/elastic/beats/compare/v8.4.0\...v8.4.1[View commits]
 
+==== Known Issue 
+
+// tag::credentials-error[]
+
+*Filebeat*
+
+Filebeat agents configured to read from S3 or SQS may return the following
+error:
+
+[source,shell]
+----
+sqs ReceiveMessage failed: operation error SQS: ReceiveMessage, https response
```

Comment:
beware that the error message might differ for some users: it's anyway most of the time related to authentication and/or region